about/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: About the Club
+---
+
+# About
+
+The University of Waterloo Amateur Radio Club is a club at the
+[University of Waterloo](https://uwaterloo.ca) for exploring
+the amateur radio hobby. The club is open to all students of the University, for both
+those who are licensed and those who are not.
+
+The club callsign is VE3UOW.
+
+# Meeting
+
+The club meets on most Friday's at 6:00PM in E2 4402. Anyone is welcome
```

Comment:
Friday's -> Fridays

Also, GH seems to be parsing this extension for syntax highlighting weirdly. :s It usually works for .md files.